README.md

```diff
@@ -27,6 +27,14 @@ metadata:
 ```
 
 ## Running the controller in a cluster
+### Prerequisites
+- go
+- git
+- kustomize.io
+- sed
+- jq
+- yq (python-yq from https://github.com/kislyuk/yq#installation, other distributions may not work)
```

Comment:
Thank a lot for this line!!!
When I need to tell people which yq is expected, I each time forget and figure out kislyuk vs mikefarah :-D